content/resources/how-to-guides/how-to-zapier-alert.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How to Alert Task to Zapier Webhook
+description: >
+  Learn how to use webhooks with InfluxDB to send alerts to Zapier
+menu:
+  resources:
+    parent: How-to guides
+weight: 102
+---
+### How To: Alert Task to Zapier Webhook
+
+### By: Tyler Nguyen
+
+If you have been a long time user of InfluxDB, the role of tasks within the platform is, without a doubt, a
+
+spotlight feature of the product. Tasks allow users to automatically execute a bit of Flux code: 
+
+downsampling, transforming, or even creating data are all things that can be done with such a powerful tool 
+
+for automation. When dealing with potentially large amounts of raw data, automation is key to an efficient 
+
+workflow. Although we know tasks are extraordinarily useful and aid the process of automatically carrying 
+
+out repetitive jobs, there is still some degree of monitoring that needs to be done on the user end. 
+
+Fortunately, there is an incredibly simple way to take the automation one step further by using webhooks 
+
+combined with Zapier, a third party web app automation service, to propagate alerts to a host of different 
+
+applications and services with an effort to mitigate the level of micromanagement users must have when 
+
+performing certain tasks.
```

Comment:
This reads more like a blog post. Documentation should have a different tone. There should be less "marketing speak." You should just provide a simple, step-by-step overview for how to do something. If conceptual information is necessary to provide context or inform the process, include it. But other language like "Although we know tasks are extraordinarily useful and aid the process of automatically carrying out repetitive jobs" can be left out.